editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -274,30 +296,35 @@ function parseObjectLiteralExpression(
     } else if (TS.isSpreadAssignment(literalProp)) {
       const subExpression = parseAttributeExpression(
         sourceFile,
+        sourceText,
         filename,
         imports,
         topLevelNames,
         propsObjectName,
         literalProp.expression,
         highlightBounds,
         alreadyExistingUIDs,
+        [],
       )
       if (isLeft(subExpression)) {
         return subExpression
       } else {
         const subExpressionValue = subExpression.value.value
-        contents.push(jsxSpreadAssignment(subExpressionValue))
+        contents.push(jsxSpreadAssignment(subExpressionValue, emptyComments))
         highlightBounds = subExpression.value.highlightBounds
         propsUsed.push(...subExpression.value.propsUsed)
         definedElsewhere.push(...subExpression.value.definedElsewhere)
       }
     }
+
+    // First prop reset after everything has been handled.
+    firstProp = false
```

Comment:
:| I think it'd be worth turning this into a const and changing the loop to another syntax which gets the index